content/hardware/08.edu/solution-and-kits/alvik/datasheet/datasheet.md

```diff
@@ -263,11 +263,16 @@ Now that you have gone through the basics of what you can do with the device, yo
 
 ### Certifications Summary
 
-| Cert          | Arduino Alvik (AKX00066) |
-| ------------- | ------------------------ |
-| **CE (EU)**   | Yes                      |
-| **UKCA (UK)** | Yes                      |
-
+| Certification       | Status (AKX00066) |
+| ------------------- | ----------------- |
+| **CE (EU)**         | Yes               |
+| **UKCA (UK)**       | Yes               |
+| **RCM (Australia)** | Yes               |
+| **REACH**           | Yes               |
+| **FCC (USA)**       | Yes               |
+| **IC (Canada)**     | Yes               |
+| **FCC (USA)**       | Yes               |
```

Comment:
```suggestion
| **RoHS (EU)**       | Yes               |
```

Reply:
@g-castronovo I noticed that FCC was duplicated and RoHS was missing. If you want I can commit the change myself.